src/rules/no-template-key.ts

```diff
@@ -1,69 +0,0 @@
-import type { Rule } from 'eslint';
-import type { AttributeToken, Loc } from 'pug-lexer';
-import { processRule } from '../utils';
-
-export default {
-  meta: {
-    type: 'problem',
-    docs: {
-      description: 'disallow `key` attribute on `<template>`',
-      categories: ['vue3-essential', 'essential'],
-      url: 'https://eslint.vuejs.org/rules/no-template-key.html',
-    },
-    fixable: undefined,
-    schema: [],
-    messages: {
-      disallow:
-        "'template' cannot be keyed. Place the key on real elements instead.",
```

Comment:
Here we lost the pug-ified version of the message

In eslint-plugin-vue it is `<template>` (see https://github.com/vuejs/eslint-plugin-vue/blob/1ce68fa9def2c7d6da7aa5020c731860d7027459/lib/rules/no-template-key.js#L30)
But in my plugin there were not `<` and `>` around `template` because pug doesn't have such brackets :thinking: 

This is only one of many examples of such messages
Is it possible to update the message for pug?

Reply:
Good point. Forking the whole rule code is a bit overkill, perhaps we could just wrap it, since that message is exposed in the rule meta object.